samples/04_gis_analysts_data_scientists/information_extraction_from_cheshire_fire_incident_reports_using_mistral_language_model.ipynb

```diff
@@ -0,0 +1,1208 @@
+{
```

Comment:
<p>To find&nbsp;<strong>precision</strong>,&nbsp;<strong>recall</strong>,&nbsp;and&nbsp;<strong>f1</strong>&nbsp;scores per label/class, we will call the model's&nbsp;<code>metrics_per_label()</code>&nbsp;method.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>